packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -377,6 +377,36 @@ import { Button, TextField } from 'ds-medicare-gov';
 
 </ThemeContent>
 
+While you can actually use Preact inside code that was written for React—and [the preact-react-app example](https://github.com/CMSgov/design-system/tree/main/examples/preact-react-app) shows you how—we've also provided an import path for you to use the Preact version of our components directly. To import the Preact versions, add `/preact` to the end of your import path like this:
```

Comment:
`-and` seems like a typo?

Reply:
In monospace you can't tell that it's an emdash, but it should look right on the doc site. I'm open to rewriting this sentence to be simpler

Reply:
`While it is possible to use Preact code inside React code` ?

Reply:
oh yeah, that's worded funny. inside a React project. whole thing maybe needs rewording. I had a hard time explaining it succinctly

Reply:
total rewrite coming